app/src/main/java/org/oppia/app/testing/ProfileChooserFragmentTestActivity.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.app.testing
+
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import org.oppia.app.profile.ProfileActivityPresenter
+import javax.inject.Inject
+
+/** Test Activity used for testing [ProfileChooserFragment] */
+class ProfileChooserFragmentTestActivity : InjectableAppCompatActivity() {
+
+  @Inject lateinit var profileActivityPresenter: ProfileActivityPresenter
```

Comment:
Since this test activity is for verifying ProfileChooserFragment, we shouldn't use ProfileActivityPresenter (that's essentially the same as using ProfileActivity). Instead, we should inflate and add ProfileChooserFragment to a test layout only used by this test activity.

Reply:
@BenHenning You mean not test fragment just add the fragment layout to activity and test?